server/datastore/mysql/operating_system_vulnerabilities.go

```diff
@@ -57,6 +59,38 @@ func (ds *Datastore) InsertOSVulnerabilities(ctx context.Context, vulnerabilitie
 	return count, nil
 }
 
+func (ds *Datastore) InsertOSVulnerability(ctx context.Context, v fleet.OSVulnerability, s fleet.VulnerabilitySource) (bool, error) {
+	if v.CVE == "" {
+		return false, nil
+	}
+
+	var args []interface{}
+
+	// statement assumes a unique index on (host_id, cve)
+	sqlStmt := `
+		INSERT INTO operating_system_vulnerabilities (
+			host_id,
+			operating_system_id,
+			cve,
+			source,
+			resolved_in_version
+		) VALUES (?,?,?,?,?)
+		ON DUPLICATE KEY UPDATE
+			operating_system_id = VALUES(operating_system_id),
+			source = VALUES(source),
+			resolved_in_version = VALUES(resolved_in_version)
```

Comment:
Update `updated_at` with `time.Now().UTC()`?

Reply:
We do that in `ds.InsertSoftwareVulnerability`, maybe let's do it for consistency and clarity?